snapshot/README.md

```diff
@@ -35,6 +35,8 @@ snapshot
 
 ###### Automate taking localized screenshots of your iOS app on every device
 
+'snapshot' generates localised iOS screenshots for different device types and languages for the App Store that are ready to be uploaded using ([`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver) is your friend). 
```

Comment:
*localized

also, the last part has strange parens. Maybe should be something like "... that can be uploaded using [`deliver`](https://github.com/fastlane/fastlane/tree/master/deliver)."